### PR TITLE
add flowBalanceTrigger

### PR DESCRIPTION
**Pull request by Peixer**

### CODE REVIEW

File: packages/components/nodes/FlowBalanceTrigger/FlowBalanceTrigger.ts [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+        this.networks = [
+            {
+                label: 'Network',
+                name: 'network',
+                type: 'options',
+                options: [...FLOWNetworks],
+                default: 'homestead'
+            }
+        ] as INodeParams[]
+        this.credentials = [...networkProviderCredentials] as INodeParams[]
```
Comment (HenryHengZJ) [inline packages/components/nodes/FlowBalanceTrigger/FlowBalanceTrigger.ts:46]: we can remove `credentials` since we are not using it here

**Review by HenryHengZJ — APPROVED** [review]
Thanks @Peixer!

**Comment by franklywatson** [comment]
HI @Peixer @HenryHengZJ - I'm from Flow blockchain team. I was looking over the other open issues, cool to see and thanks for kicking it off! Question I had was how was this trigger tested? I'm curious if it's something I could test out. There are also other token types that would ideally need to be added which we could help with. Is there a specific process for proposing PR changes that we should know about?

**Comment by Peixer** [comment]
Hey @franklywatson, you can check out this post https://blog.outerbridge.io/blog/flow-outerbridge, there is a template you can copy and paste if I'm not mistaken. @HenryHengZJ can help you with the PR template :)

**Comment by HenryHengZJ** [comment]
@franklywatson feel free to propose submit PR as long as the code is self-explanatory or with clear description if code has more changes, there isn't any strict template to follow. You can also take a look at the [docs](https://docs.outerbridge.io) to see how you can create or add functionalities. Feel free to reach out in [Discord](https://discord.com/invite/Y9VE4ykPDJ) too if you have any questions

**Pull request merged**